### PR TITLE
updated xref for paroxysmal nocturnal hemoglobinuria

### LINKED ISSUE #4232: MONDO:0100244  paroxysmal nocturnal hemoglobinuria (vs. acquired form: MONDO:0100245)
**Mondo term (ID and Label):**
MONDO:0100245, acquired paroxysmal nocturnal hemoglobinuria seems to have X-refs that belong to the parent concept; and maybe some other X-refs need to be reviewed and maybe moved?

**Xref that should be fixed (ID and label):**
SCTID 1963002
HP:0004818

Also ORPHA:447 appears to be equivalent to the OMIMPS:300818. 

**Your nano-attribution (ORCID)**
If you don't have an ORCID, you can sign up for one [here](https://orcid.org/)

**Other comments:**

### PR DESCRIPTION
**Pull request by sabrinatoro**
Fixes #4232

@Nicole: I wasn't sure what to do with the HP xref as it refers to a phenotype (and therefore not a disease). Please advice on this. Thank you.

### CODE REVIEW

File: src/ontology/mondo-edit.properties [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#Tue Jan 04 12:54:28 PST 2022
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.properties:1]: did you mean to edit this file?

Comment (sabrinatoro) [reply src/ontology/mondo-edit.properties:1]: no I didn't... I don't know how it was edited in the first place...

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -441658,7 +441660,7 @@ synonym: "PNH" EXACT ABBREVIATION [Orphanet:447]
 xref: DOID:0060284 {source="MONDO:equivalentTo"}
 xref: GARD:0007337 {source="MONDO:equivalentTo"}
 xref: HGNC:8957 {source="GARD:0007337"}
-xref: HP:0004818 {source="MONDO:otherHierarchy", source="DOID:0060284"}
+xref: HP:0004818 {source="MONDO:subClassOf", source="MONDO:otherHierarchy", source="DOID:0060284"}
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:441663]: I believe the MONDO:subClassOf is only meant for Mondo classes, but I understand your thought process. I think in this case, we would just delete the HPO xref here and move it to the parent class (MONDO_01002443 'paroxysmal nocturnal hemoglobinuria'). Add a seeAlso to the ticket and then we'll have the history of this HPO xref being on this term.

**Review by nicolevasilevsky — COMMENTED** [review]
see comments

File: src/ontology/mondo-edit.properties (hunk 1 above)
Comment (sabrinatoro) [reply src/ontology/mondo-edit.properties]: fixed

**Review by nicolevasilevsky — APPROVED** [review]

**Review by nicolevasilevsky — APPROVED** [review]

**Pull request merged**